README.md

```diff
@@ -1,3 +1,17 @@
 # Literature Reviews
 
 The goal of this repo is to house markdown documents exploring proposed language and library additions to the Elm language.
+
+
+## Gathering feedback before submitting
+
+It's often helpful to get feedback on your concept before getting into the detail required for a literature review.
+You may open an issue on this repo to start a high-level discussion, with the goal of eventually creating a literature review pull request. In addition you can post a message to the [elm-discuss mailing list](https://groups.google.com/forum/#!forum/elm-discuss) or chat to the folks over at [Slack](https://elmlang.herokuapp.com/).
+
+## The process
+
+* Fork the [the elm-community/literature-reviews repo](https://github.com/elm-community/literature-reviews)
+* Copy `TEMPLATE.md` to `text/my-feature-name/README.md`
```

Comment:
still, why `text`? It's not like this repo will contain thing *other* than literature reviews, right? It's premature namespacing, and if we need it in the future we can always shove them all into a subdirectory.